docs/getting_started.md

```diff
@@ -1,58 +1,122 @@
+
+In addition to **serve** you can use:
+
+- **build:** Runs a single build and exits. This is most useful if your build also
+  generates output to your source directory. With `--output <dirname>` this
   also creates a merged output directory with all sources and generated assets.
-- `watch` - like `build` but reruns after file changes. With
+
+- **watch:** Like `build` but reruns after file changes. With
   `--output <dirname>` the merged output directory will be kept up to date with
   changes. This can be used to keep the outputs updated for use with another
   filed-based development server.
-- `test` - Creates an output directory and runs `pub run test` within it.
 
-## Switching to Dart2Js
+- **test:** Creates an output directory and runs `pub run test` within it.
 
-By default `build_web_compilers` will use DDC. Switch to Dart2JS by creating a
-`build.yaml` file with a `target` that matches the name of your packages.
+## Switching to dart2js
+
+By default `build_web_compilers` uses dartdevc. To switch to dart2js,
+create a `build.yaml` file with a `target` that matches the name of your packages.
 
 ```yaml
 targets:
   my_package_name:
     builders:
-      dart_web_compilers|entrypiont:
+      dart_web_compilers|entrypoint:
         options:
           compiler: dart2js
 ```
 
+<!--PENDING: Why does this show a `builders` line? Is that necessary for
+dart2js?-->
+
+
 ## Compatibility with other packages
 
-### Upgrading from Transformers
+### Upgrading from transformers
 
-`build_runner` will only run Builders which are published with a `build.yaml`
-file. Legacy `Transformers` cannot be run by `build_runner`. Check with the
-packages that provide Transformers you are using for versions published with a
+`build_runner` can only run Builders that are published with a `build.yaml`
+file; it can't use legacy `Transformers`. If your pubspec lists transformers,
+switch to a version of the transformer-containing package that has a
 `build.yaml` file.
 
 ### Upgrading from manual `build.dart` files
 
 Older versions of `build_runner` were designed to run with manually written
 build scripts referencing the Builders available in the local package or in
 dependencies. This pattern can still be used when customization is needed
-outside of `build.yaml`, but we recommend most users use the generated build
-script with `pub run build_runner` since it will be kept up to date with changes
-in the build packages. Check with the packages that provide Builders you are
-using for versions published with a `build.yaml` file.
+outside of `build.yaml`, but we recommend using the generated build script
+with `pub run build_runner`, because it will be kept up to date with changes
+in the build packages. If your pubspec lists builders, switch to a version of
+the builder-containing package that has a `build.yaml` file.
+
+
+## Troubleshooting
+
+<!-- summarize here. -->
+
+<!-- add something about symptoms if you don't remove transformers -->
```

Comment:
There shouldn't be any symptoms for including or removing the config.

Reply:
We could add a warning if we still see a transformers section or something, idk